db/cofed.sql

```diff
@@ -46,3 +49,63 @@ GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO root;
 GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO root;
 
 GRANT ALL PRIVILEGES ON DATABASE cofed TO root;
+
+-- dummy data below
+
+INSERT INTO coops
+VALUES
+    (1, 'test@gmail.com', 'password', 'Test', '123456789',
+        'Berkeley, CA', 37.8715, -123.2730, 'test.com', 'test', 'test', 'test', 'test',
+        'test');
+
+INSERT INTO coops
+VALUES
+    (2, 'test2@gmail.com', 'password2', 'Test2', '123456789',
+        'Berkeley, CA2', 37.8715, -123.2730, 'test.com', 'test2', 'test2', 'test2', 'test2',
+        'test2');
+
+INSERT INTO tags
+VALUES
+    (1, 'f');
+
+INSERT INTO tags
+VALUES
+    (2, 's');
+
+INSERT INTO tags
+VALUES
+    (3, 't');
+
+INSERT INTO tags
+VALUES
+    (4, 'NOOT IN COOP 1 TAG');
+
+INSERT INTO tags
+VALUES
+    (5, 'NO COOP 1 TAG');
+
+
+INSERT INTO coop_tags
+VALUES
+    (1, 1, 1);
+
+INSERT INTO coop_tags
+VALUES
+    (2, 1, 2);
+
+INSERT INTO coop_tags
+VALUES
+    (3, 1, 3);
+
+INSERT INTO coop_tags
+VALUES
+    (4, 2, 4);
+
+INSERT INTO coop_tags
+VALUES
+    (5, 2, 5);
+
+INSERT INTO coop_tags
+VALUES
+    (6, 2, 3);
```

Comment:
for these queries, do not explicitly insert the `id`, otherwise the SERIAL logic will not work!! for instance, the first two `coops` queries should be
```sql
INSERT INTO coops
    (email, pass, coop_name, phone_number, addr, latitude, longitude, website, mission_statement, description_text, profile_pic, insta_link, fb_link)
VALUES
    ('test@gmail.com', 'password', 'Test', '123456789', 'Berkeley, CA', 37.8715, -123.2730, 'test.com', 'test', 'test', 'test', 'test', 'test'),
    ('test2@gmail.com', 'password2', 'Test2', '123456789', 'Berkeley, CA2', 37.8715, -123.2730, 'test.com', 'test2', 'test2', 'test2', 'test2', 'test2');
```

please do this with the tags as well